src/data/showcase/elian-codes.json

```diff
@@ -1,6 +1,6 @@
 {
-	"title": "Elian Codes",
```

Comment:
Revert this file and we should be all set 👍

Reply:
ack! Will do. Thanks!